assets/keymaps/vim.json

```diff
@@ -324,17 +326,8 @@
       ],
       "u": "editor::Undo",
       "ctrl-r": "editor::Redo",
-      "/": "vim::Search",
-      "?": [
-        "vim::Search",
-        {
-          "backwards": true
-        }
-      ],
```

Comment:
This is intentional, yes?

Reply:
haha, absolutely not!

I tried moving them into the shared section too, but then hitting / breaks visual selection, so more work needed; and I mis-reverted them